hack/testing/test-viaq-data-model.py

```diff
@@ -25,6 +25,16 @@ def empty(thing):
             return True
     return False
 
+def dictCompare(str1, str2):
+    s1_obj = json.loads(s1)
+    s2_obj = json.loads(s2)
```

Comment:
```python
    return s1_obj == s2_obj
```
comparison of `dict` in python is recursive https://docs.python.org/3/reference/expressions.html#comparisons
what it doesn't say is that if the value is a `dict`, it will do a deep comparison of those:
```
>>> bar = {"sub":{7:8,"ary":[1,2,3,4],"subsub":{"a":"b"}},5:6, 3:4, 1:2}
>>> foo = {1:2, 5:6, 3:4, "sub":{"subsub":{"a":"b"},"ary":[1,2,3,4],7:8}}
>>> foo == bar
True
```